app/src/main/res/values/strings.xml

```diff
@@ -1,10 +1,12 @@
 <resources>
     <string name="app_name">FOrder</string>
+    <string name="mgs_product_time_out">This product is time out, you can not order this product!</string>
     <string name="mgs_this_order">Would you like this order?</string>
     <string name="mgs_order_all">Would you like order all?</string>
     <string name="mgs_delete_product">Would you like delete this product?</string>
     <string name="action_yes">Yes</string>
     <string name="action_no">No</string>
+    <string name="action_ok">Ok</string>
```

Comment:
WARNING: [ButtonCase] The standard Android way to capitalize Ok is "OK" (tip: use `@android:string/ok` instead)
 The standard capitalization for OK/Cancel dialogs is "OK" and "Cancel". To ensure that your dialogs use the standard strings, you can use the resource strings @android:string/ok and @android:string/cancel.